.circleci/config.yml

```diff
@@ -1345,6 +1345,7 @@ workflows:
         - not: << pipeline.parameters.generate_revenuecatui_snapshots >>
     jobs:
       - lint
+      - run-test-ios-13
```

Comment:
do we want to keep this?

Reply:
I had it there for testing - it's removed now 😄